File: P2N_Pet_FrontEnd/P2N-Pet/projects/admin/src/app/models/customer.ts
export class Customer{
    public Id: number;
    public UserName: string;
    public Name: string;
    public Birthday: string;
    public Address: string;
    public Phone: string;
    public Email: string;
    public StatusText: string;
    public CreateUserName: string;
    public CreateDate: string;
    public UpdateUserName: string;
    public UpdateDate: string;
}

export class CustomerCondition {
    Name: string;
    Phone: string;
    Email: string;
    Status: string;
  
    constructor() {
      this.Name = '';
      this.Phone = '';
      this.Email = '';
      this.Status = '0';
    }
  }

File: P2N_Pet_BackEnd/P2N_Pet_API/P2N_Pet_API/Database/PetShopModels/Customer.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace P2N_Pet_API.Database.PetShopModels
{
    public partial class Customer
    {
        public Customer()
        {
            Orders = new HashSet<Order>();
        }

        public ulong Id { get; set; }
        public ulong? Userid { get; set; }
        public string Name { get; set; }
        public DateTime? Birthday { get; set; }
        public string Address { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public int? Status { get; set; }
        public ulong? Createuser { get; set; }
        public DateTime? Createdate { get; set; }
        public ulong? Updateuser { get; set; }
        public DateTime? Updatedate { get; set; }

        public virtual User User { get; set; }
        public virtual ICollection<Order> Orders { get; set; }
    }
}
